stock_orderpoint_move_link/__manifest__.py

```diff
@@ -1,13 +1,13 @@
-# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L., Ecosoft
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 
 {
     "name": "Stock Orderpoint Move Link",
     "summary": "Link Reordering rules to stock moves",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "license": "LGPL-3",
     "website": "https://github.com/stock-logistics-warehouse",
-    "author": "Eficent, "
+    "author": "Eficent, Ecosoft, "
```

Comment:
A migration does not make you author of a module, you should put yourself in the contributors fragment of the readme.